### PR TITLE
mmtc: init at 0.2.6

### PR DESCRIPTION
**Pull request by figsoda**
###### Motivation for this change
mmtc is a minimal mpd terminal client that aims to be simple yet highly configurable 

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by figsoda** [comment]
Result of `nixpkgs-review pr 105702` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>mmtc</li>
  </ul>
</details>

**Comment by figsoda** [comment]
Ofborg didn't seem to like it. 

> This PR does not cleanly list package outputs after merging.

What does it mean? What can I do to fix it?

File: pkgs/applications/audio/mmtc/default.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  RUSTFLAGS = "";
+
+  cargoSha256 = "1g3512gi4z3hykr9f04rslm6aymk8lph6g851qwq0aip8jjqgz5h";
+
+  meta = with stdenv.lib; {
+    description =
+      "Minimal mpd terminal client that aims to be simple yet highly configurable";
+    homepage = "https://github.com/figsoda/mmtc";
+    license = licenses.mpl20;
+    maintainers = [ ];
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/mmtc/default.nix:36]: Missing maintainers and platforms.

**Comment by figsoda** [comment]
Ofborg still says the same thing. Does it have something to do with `nixpkgs-mozilla`? Since it depends on the nightly version of rust.

File: pkgs/applications/audio/mmtc/default.nix [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{ callPackage, fetchFromGitHub, rustPlatform, stdenv }:
+
+let
+  rust = (let
+    mozilla = fetchFromGitHub {
+      owner = "mozilla";
+      repo = "nixpkgs-mozilla";
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/mmtc/default.nix:7]: I don't think this is acceptable in nixpkgs.

**Review by SuperSandro2000 — COMMENTED** [review]
Please squash your mmtc related commits together.

File: pkgs/applications/audio/mmtc/default.nix (hunk 2 above)
Comment (figsoda) [reply pkgs/applications/audio/mmtc/default.nix:7]: > I don't think this is acceptable in nixpkgs.

I guess that's why, I will close this then

**Pull request closed without merging**